outlines/using-packages.md

```diff
@@ -0,0 +1,34 @@
+# Using packages
+
+1. Packages vs modules
+  - Packages are useful for sharing content between apps and users
+  - It's usually better to leverage someone else's work and use a package if it does what you need
+2. Npm vs Atmosphere
+  1. Npm packages:
```

Comment:
I feel like these could be the first sections of the "Atmosphere" and "NPM" sections below? Unless there's a feature table or something.